db/migrate/20160829134746_remove_atomic_container_providers.rb

```diff
@@ -0,0 +1,47 @@
+class RemoveAtomicContainerProviders < ActiveRecord::Migration[5.0]
```

Comment:
Code-wise I am good with this migration aside from the minor comments elsewhere and aside from my general question about if we even *should* have this migration.